src/preferences/dialog/dlgpreflibrary.cpp

```diff
@@ -277,6 +277,8 @@ void DlgPrefLibrary::slotRemoveDir() {
     } else if (removeMsgBox.clickedButton() == deleteAllButton) {
         removalType = Library::RemovalType::PurgeTracks;
     } else {
+        // prevent warning depending on build type
```

Comment:
The comment requires some thinking. Why not simply
// Only used in DEBUG_ASSERT